edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.DiagnosticsComponent/DiagnosticConfig.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.Azure.Devices.Edge.Agent.DiagnosticsComponent
```

Comment:
Should we rename this namespace to `Microsoft.Azure.Devices.Edge.Agent.Diagnostics`? I'm not sure the `Component` buys us anything.